lib/cucumber/runtime.js

```diff
@@ -7,15 +7,12 @@ function Runtime(configuration) {
     start: function start(callback) {
       if (typeof(callback) !== 'function')
         throw new Error(Runtime.START_MISSING_CALLBACK_ERROR);
-      var isStrictRequested;
-      try {
-        isStrictRequested    = configuration.isStrictRequested();
-      } catch(e) {
-        isStrictRequested    = false;
-      }
+
+      var isStrictRequested  = configuration.isStrictRequested && configuration.isStrictRequested();
```

Comment:
Please revert this change. Its unrelated and I'd prefer to have it dealt with separately